.github/workflows/ci.yml

```diff
@@ -746,3 +763,94 @@ jobs:
           name: test report pt (${{ matrix.config }}, ${{ matrix.suite }}, ${{ matrix.jdk }})
           path: testing/trino-product-tests/target/reports/**/testng-results.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
+
+  delta-lake-databricks-pt:
+    runs-on: ubuntu-latest
+    name: pt (${{ matrix.config }}, ${{ matrix.suite }}, ${{ matrix.jdk }})
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - default
+        suite:
+          - suite-delta-lake-databricks
```

Comment:
Does this needs to be a separate job? Could it use a similar condition to this?
```
          - suite: suite-azure
            ignore exclusion if: >-
              ${{ needs.build-pt.outputs.have_azure_secrets == 'true' }}
```

Reply:
> Does this needs to be a separate job?

The Databricks Delta Lake product test environment relies on AWS & Databricks infrastructure.

In order to connect to AWS, we make use of the environment variables:

- `AWS_ACCESS_KEY_ID`
- `AWS_SECRET_ACCESS_KEY`

which are being used by the default AWS credentials provider chain.


These security credentials are not the same ones as the ones corresponding to the Github CI runner. This is why I am thinking that their usage should be made in a different job. Please advise whether there is a cheaper way of solving the above mentioned situation.

Reply:
Maybe in `EnvSinglenodeDeltaLakeDatabricks` we could redefine these env vars from the prefixed ones. But it shouldn't block this PR, we can address this later.

Reply:
> Maybe in EnvSinglenodeDeltaLakeDatabricks we could redefine these env vars from the prefixed ones.

This is definitely cheaper than my approach. I'd say that it makes more sense. I'll do the modifications right away.